README.md

```diff
@@ -8,7 +8,7 @@ decoder), called `libjxl`.
 JPEG XL is in the final stages of standardization and its codestream format is
 frozen.
 
-The libraries API, command line options and tools in this repository are subject
+The libraries API, command line options, and tools in this repository are subject
```

Comment:
Now that I see this, shouldn't that be `The library API` or maybe `The library's API`?

Reply:
Thanks.  I changed it to "library API".